src/components/modals/HintModal.tsx

```diff
@@ -0,0 +1,28 @@
+import { referenceUrl } from '../../lib/words'
+import { BaseModal } from './BaseModal'
+
+type Props = {
+  isOpen: boolean
+  handleClose: () => void
+  verseText: string
+  referenceUrl: string
+}
+
+export const HintModal = ({ isOpen, handleClose, verseText }: Props) => {
+  return (
+    <BaseModal title="Hint" isOpen={isOpen} handleClose={handleClose}>
+      <img src="OpenQuote.png" className="absolute top-8 left-5 w-10" />
```

Comment:
these images should be in the `src/img` folder.

instead of giving a path, import the path into the file.

look at the About page for an example

Reply:
Also, a portion of this quote image is cut off in dark mode. Give it a z-index to fix that issue.

Reply:
I moved the images to the img folder, and now it cant seem to find them haha. Any idea why that would be? Ive tried every path to them and nothing has worked.

Reply:
You're import will look like this: `import gitLogo from './../img/github-mark.png'`

Reply:
got it! thanks. Also, where do you see the cutoff? It appears to me they are both fully there until I zoom in 300%.

Reply:
![Screenshot 2023-02-23 at 5 05 47 PM](https://user-images.githubusercontent.com/70766937/221051413-e9a35b88-a2f7-4f68-b5ea-27aa3ca1345f.png)
See how you can't see the bottom of the quote? A z-index should fix this though!

Reply:
oh ok. Mobile stuff!! Lemme fix that thank you!